irohad/ametsuchi/wsv_query.hpp

```diff
@@ -40,6 +40,35 @@ namespace iroha {
           std::vector<std::shared_ptr<shared_model::interface::Peer>>>
       getPeers() = 0;
 
+      // ToDo?
```

Comment:
I would suggest either to create a task and format as `// TODO(kuvaldini) getDomains query` or remove.

Reply:
that was WIP, okay #997